vkcs/vpnaas/resource_site_connection.go

```diff
@@ -161,6 +163,12 @@ func ResourceSiteConnection() *schema.Resource {
 				ValidateDiagFunc: networking.ValidateSDN(),
 				Description:      "SDN to use for this resource. Must be one of following: \"neutron\", \"sprut\". Default value is project's default SDN.",
 			},
+			"traffic_selector_ep_merge": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     true,
```

Comment:
should default be true in fact?

Reply:
Yes, I check it in sprut repo. Also I checked that this parameter can be updated without recreating the resource.